test/e2e/data/infrastructure-docker/main/clusterclass-quick-start-runtimesdk-v1beta1.yaml

```diff
@@ -98,6 +98,7 @@ spec:
       kubeadmConfigSpec:
         clusterConfiguration:
           apiServer:
+            timeoutForControlPlane: 20m
```

Comment:
Some easy e2e test coverage. This would fail without the fix, because the KubeadmConfig that KCP generates only have InitConfiguration or JoinConfiguration defined, but not both